### PR TITLE
Backport missing PRs from Bitcoin 0.14 - Part 5

### PR DESCRIPTION
**Pull request by codablock**
Continuation of https://github.com/dashpay/dash/pull/1770

### CODE REVIEW

File: src/policy/fees.cpp [hunk 1]
```diff
@@ -533,51 +442,38 @@ CFeeRate CBlockPolicyEstimator::estimateSmartFee(int confTarget, int *answerFoun
...
 
-void CBlockPolicyEstimator::Read(CAutoFile& filein)
+void CBlockPolicyEstimator::Read(CAutoFile& filein, int nFileVersion)
 {
     int nFileBestSeenHeight;
     filein >> nFileBestSeenHeight;
     feeStats.Read(filein);
-    priStats.Read(filein);
     nBestSeenHeight = nFileBestSeenHeight;
+    if (nFileVersion < 139900) {
```
Comment (UdjinM6) [inline src/policy/fees.cpp]: 120300

File: src/txmempool.cpp [hunk 2]
```diff
@@ -1029,7 +1029,7 @@ CTxMemPool::WriteFeeEstimates(CAutoFile& fileout) const
 {
     try {
         LOCK(cs);
-        fileout << 120000; // version required to read: 0.12.00 or later
+        fileout << 120201; // version required to read: 0.12.00 or later
```
Comment (UdjinM6) [inline src/txmempool.cpp]: 120300

File: Makefile.am [hunk 3]
```diff
@@ -40,6 +40,11 @@ OSX_PLIST=$(top_builddir)/share/qt/Info.plist #not installed
 OSX_QT_TRANSLATIONS = da,de,es,hu,ru,uk,zh_CN,zh_TW
 
 DIST_DOCS = $(wildcard doc/*.md) $(wildcard doc/release-notes/*.md)
+DIST_CONTRIB = $(top_srcdir)/contrib/bitcoin-cli.bash-completion \
```
Comment (UdjinM6) [inline Makefile.am]: this block should be dashified

File: src/qt/sendcoinsdialog.cpp [hunk 4]
```diff
@@ -700,15 +714,17 @@ void SendCoinsDialog::updateGlobalFeeVariables()
...
     {
-        nTxConfirmTarget = defaultConfirmTarget - ui->sliderSmartFee->value();
+        int nConfirmTarget = ui->sliderSmartFee->maximum() - ui->sliderSmartFee->value() + 1;
         payTxFee = CFeeRate(0);
 
         // set nMinimumTotalFee to 0 to not accidentally pay a custom fee
         CoinControlDialog::coinControl->nMinimumTotalFee = 0;
+
+        // show the estimated reuquired time for confirmation
+        ui->confirmationTargetLabel->setText(GUIUtil::formatDurationStr(nConfirmTarget*600)+" / "+tr("%n block(s)", "", nConfirmTarget));
```
Comment (UdjinM6) [inline src/qt/sendcoinsdialog.cpp]: s/600/Params().GetConsensus().nPowTargetSpacing/

**Comment by codablock** [comment]
Pushed review fixes

**Pull request merged**